playground/src/file-upload/file-upload.scss

```diff
@@ -0,0 +1,85 @@
+@import '_variables.scss';
+@import '_functions.scss';
+@import 'components/atoms/_text.scss';
+
+.file-upload {
+  &-dropzone {
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: center;
+    align-items: center;
+    justify-content: center;
+    height: 250px;
+    padding: 40px;
+    border: 2px dashed color('silver');
+    border-radius: 4px;
+    cursor: pointer;
+
+    @include txt;
+    @include txt-2;
+
+    &.dz-drag-hover {
+      background: transparentize(color('silver'), 0.8);
+    }
+
+    .dz-info {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+
+      &:before {
+        content: '';
+        display: block;
+        width: 30px;
+        height: 30px;
+        margin: auto;
+        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='30' width='30' viewBox='0 0 100 100'%3E%3Cpath style='text-indent:0;text-transform:none;block-progression:tb' d='M13 22c-2.753 0-5 2.247-5 5v46c0 2.753 2.247 5 5 5h74c2.753 0 5-2.247 5-5V27c0-2.753-2.247-5-5-5H13zm0 2h74c.43 0 .852.093 1.219.25L52.406 57.125c-1.236 1.135-3.576 1.135-4.812 0L11.78 24.25c.367-.157.79-.25 1.219-.25zm-2.688 1.656l26.376 24.157-26.376 24.53A3.03 3.03 0 0110 73V27c0-.493.11-.937.313-1.344zm79.376 0c.203.407.312.851.312 1.344v46c0 .486-.114.94-.313 1.344L63.314 49.813l26.374-24.157zm-51.5 25.532l8.062 7.406c2.075 1.904 5.425 1.904 7.5 0l8.063-7.406L88.25 75.75c-.377.168-.805.25-1.25.25H13c-.445 0-.873-.082-1.25-.25l26.438-24.562z' overflow='visible' color='%23000'/%3E%3C/svg%3E");
```

Comment:
This icon is doubled - and should be more generic than the envelope (or be configurable)